failing 1 unit test on local:
This is probably just an existing problem, unit tests through GH actions work just fine
```
marqo.errors.BadRequestError: BadRequestError: Problem vectorising query. Reason: Unable to load model=ViT-B/16 on device=cpu with normalization=True. If you are trying to load a custom model, please check that model_properties={'name': 'ViT-B/16', 'dimensions': 512, 'notes': 'CLIP ViT-B/16', 'type': 'clip'} is correct and Marqo has access to the weights file.
```

```
============================================== short test summary info ==============================================
FAILED tests/tensor_search/test_add_documents.py::TestAddDocuments::test_mappings_arent_updated_images - marqo.errors.BadRequestError: BadRequestError: Problem vectorising query. Reason: Unable to load model=ViT-B/16 ...
```

--- a/src/marqo/tensor_search/on_start_script.py
+++ b/src/marqo/tensor_search/on_start_script.py
@@ -182,8 +182,8 @@ def _preload_model(model, content, device):
             )
         except KeyError as e:
             raise errors.EnvVarError(
-                f"Your custom model {model} is missing either `model_name` or `model_properties`."
-                f"""To add a custom model, it must be a dict with keys `model` and `model_properties` as defined in `https://marqo.pages.dev/0.0.20/Models-Reference/bring_your_own_model/`"""
+                f"Your custom model {model} is missing either `model` or `model_properties`."
+                f"""To add a custom model, it must be a dict with keys `model` and `model_properties` as defined in `https://marqo.pages.dev/0.0.20/Advanced-Usage/configuration/#configuring-preloaded-models`"""
             ) from e
 
 

